### PR TITLE
(XW-1735) Escape content of wrap-me helper by default

### PR DESCRIPTION
**Pull request by Grunny**
## Links

* https://wikia-inc.atlassian.net/browse/XW-1735

## Description

Make sure the wrap-me helper escapes the content passed to it by default. See ticket
for details.

## Reviewers

/cc @Wikia/x-wing @Wikia/west-wing

### CODE REVIEW

File: front/main/app/components/discussion-welcome.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default Ember.Component.extend({
 	wasSeen: Boolean(localStorage.getItem('discussionWelcomeMessageSeen')),
 
 	guidelinesLink: wrapMeHelper.compute([
-		Ember.Handlebars.Utils.escapeExpression(i18n.t('main.guidelines-link-title', {ns: 'discussion'}))
+		i18n.t('main.guidelines-link-title', {ns: 'discussion'})
```
Comment (rogatty) [inline front/main/app/components/discussion-welcome.js]: /cc @bkoval @rafalkalinski

File: front/main/app/helpers/wrap-me.js [hunk 2]
```diff
@@ -22,10 +26,14 @@ import Ember from 'ember';
...
 
 export default Helper.helper((params, options) => {
-	const content = params[0] || '';
-	let tagName = 'span',
+	let content = params[0] || '',
+		tagName = 'span',
 		className = '';
 
+	if (!options.allowRawHTML) {
+		content = Handlebars.Utils.escapeExpression(content);
```
Comment (rogatty) [inline front/main/app/helpers/wrap-me.js]: Do we really need this option it if it's not used anywhere? Why was it used before?

Comment (Grunny) [inline front/main/app/helpers/wrap-me.js]: I went back and forth on whether to provide the option or not. I hedged and provided it as it made sure it was available when needed. But, I'll go back and just always escape. :)

Comment (Grunny) [inline front/main/app/helpers/wrap-me.js]: Done.

**Comment by rogatty** [comment]
All right, I get it now. The issue was that before this PR devs had to manually escape strings in their components when using the helper and some didn't do it. Now it will be done for them automatically.

LGTM!

**Comment by dianafa** [comment]
LGTM

**Pull request merged**